jwst/ami/ami_analyze_step.py

```diff
@@ -32,8 +34,18 @@ def process(self, input):
         # Retrieve the parameter values
         oversample = self.oversample
         rotate = self.rotation
-        self.log.info('Oversampling factor = %d', oversample)
-        self.log.info('Initial rotation guess = %g deg', rotate)
+
+        # pull out parameters that are strings and change to floats
+
+        psf_offset = self.set_psf_offset_find_rotation.split(' ')
+        psf_offset_find_rotation = (float(psf_offset[0]), float(psf_offset[1]))
+        rotsearch = self.set_rotsearch_d.split(' ')
+        rotsearch_parameters= [float(rotsearch[0]), float(rotsearch[1]), float(rotsearch[2])]
+
+        self.log.info(f'Oversampling factor =  {oversample}')
+        self.log.info(f'Initial rotation guess = {rotate} deg')
+        self.log.info(f'Initial values to use for psf offset = {psf_offset_find_rotation}')
+        self.log.info(f'Initial values to use for rotation search {rotsearch_parameters}')
```

Comment:
This logging already happens when the step is initialized.  It's not as nicely formatted, but it is there.  It then gets logged again, each on its own line:

```
2020-12-17 14:09:38,885 - stpipe.Ami3Pipeline.ami_analyze - INFO - Step ami_analyze parameters are: {'pre_hooks': [], 'post_hooks': [], 'output_file': None, 'output_dir': None, 'output_ext': '.fits', 'output_use_model': False, 'output_use_index': True, 'save_results': False, 'skip': False, 'suffix': None, 'search_output_file': True, 'input_dir': '/private/var/folders/jg/by5st33j7ps356dgb4kn8w900001n5/T/pytest-of-jdavies/pytest-30/test_niriss_ami3_run_pipeline0', 'oversample': 3, 'rotation': 1.49}
2020-12-17 14:09:38,885 - stpipe.Ami3Pipeline.ami_analyze - INFO - Oversampling factor = 3
2020-12-17 14:09:38,885 - stpipe.Ami3Pipeline.ami_analyze - INFO - Initial rotation guess = 1.49 deg
```

Would be nice if we could make `stpipe` print these to be more readable by using `pprint.pformat` or similar.

Fine to keep as is though I guess, as it was there originally.